### ISSUE TITLE
EPIC - Develop and optimize a model for the classification of retina scans

### ISSUE THREAD

**Issue by db-nv** [db-nv/retina#1]
The focus of this epic is on the development and optimization of a model that can classify retina scans according to the severity of diabetic retinopathy.

Progress:

1. Initial model architecture, EfficientNet, selected and implemented
2. Training and optimization of model underway, utilizing MSE and CrossEntropy loss functions
3. Preliminary results promising, with CrossEntropy yielding 86% accuracy
